### PR TITLE
chore: redo structure for cards

### PR DESCRIPTION
**Pull request by rebelchris**
## Changes

Big PR, but most stuff is actually just swapping file names 😅

## Events

Did you introduce any new tracking events?

## Experiment

Did you introduce any new experiments?

## Manual Testing

> [!CAUTION]
> Please make sure existing components are not breaking/affected by this PR

### Preview domain
https://chore-structure-redo.preview.app.daily.dev

### CODE REVIEW

**Review by omBratteng — APPROVED** [review]

**Review by nensidosari — APPROVED** [review]
Great work!! nothing blocking on my comments, just further improvements if we want to do.

File: packages/shared/src/components/FeedItemComponent.tsx [hunk 1]
```diff
@@ -1,34 +1,37 @@
 import React, { FunctionComponent, ReactElement } from 'react';
 import dynamic from 'next/dynamic';
 import { FeedItem } from '../hooks/useFeed';
-import { ArticlePostCard } from './cards/ArticlePostCard';
-import { ArticlePostList } from './cards/list/ArticlePostList';
-import { PlaceholderCard } from './cards/PlaceholderCard';
-import { PlaceholderList } from './cards/list/PlaceholderList';
+import { PlaceholderGrid } from './cards/placeholder/PlaceholderGrid';
+import { PlaceholderList } from './cards/placeholder/PlaceholderList';
```
[the remark this answers is not in the document]
Comment (rebelchris) [reply packages/shared/src/components/FeedItemComponent.tsx:5]: This is still a open discussion point on web agenda if we should bother with barrel imports or not (it comes with a lot of downsides as well with conflicting cycle imports)

File: packages/shared/src/components/cards/SimilarPosts/PostEngagementCounts.tsx [hunk 2]
```diff
@@ -1,6 +1,6 @@
 import React, { ReactElement } from 'react';
 import classNames from 'classnames';
-import { separatorCharacter } from '../common';
+import { separatorCharacter } from '../common/common';
```
[the remark this answers is not in the document]
Comment (nensidosari) [reply packages/shared/src/components/cards/SimilarPosts/PostEngagementCounts.tsx:3]: Yeah, I'll save this, maybe even place it in discussion items so whenever we find time we see if we have a better idea for it.

File: packages/shared/src/components/FeedItemComponent.tsx (hunk 1 above)
[the remark this answers is not in the document]
Comment (nensidosari) [reply packages/shared/src/components/FeedItemComponent.tsx:5]: Well with cycle imports I think if we are careful about what we export, then we should be good with it. But I'm also a bit 2 sided about these, so let's keep it as something we try to discuss when we find the time as well.

**Pull request merged**